docs/experiments-covid.md

```diff
@@ -50,6 +74,86 @@ TODO:
 + Run query expansion.
 + Run different fusion techniques.
 
+Exact commands for replicating these runs are found [further down on this page](experiments-covid.md#round-1-replication-commands).
+
+## Round 2: Replication Commands
+
+Here are the replication commands for the individual runs:
+
+```bash
+wget https://www.dropbox.com/s/wxjoe4g71zt5za2/lucene-index-cord19-abstract-2020-05-01.tar.gz
+tar xvfz lucene-index-cord19-abstract-2020-05-01.tar.gz
+
+target/appassembler/bin/SearchCollection -index lucene-index-cord19-abstract-2020-05-01 \
+ -topicreader Covid -topics src/main/resources/topics-and-qrels/topics.covid-round2.xml -topicfield query+question -removedups \
+ -bm25 -hits 10000 -output runs/anserini.covid-r2.abstract.qq.bm25.txt -runtag anserini.covid-r2.abstract.qq.bm25.txt
+
+target/appassembler/bin/SearchCollection -index lucene-index-cord19-abstract-2020-05-01 \
+ -topicreader Covid -topics src/main/resources/topics-and-qrels/topics.covid-round2-udel.xml -topicfield query -removedups \
+ -bm25 -hits 10000 -output runs/anserini.covid-r2.abstract.qdel.bm25.txt -runtag anserini.covid-r2.abstract.qdel.bm25.txt
+
+eval/trec_eval.9.0.4/trec_eval -c -M1000 -m all_trec src/main/resources/topics-and-qrels/qrels.covid-round1.txt runs/anserini.covid-r2.abstract.qq.bm25.txt | egrep '(ndcg_cut_10 |recall_1000 )'
+eval/trec_eval.9.0.4/trec_eval -c -M1000 -m all_trec src/main/resources/topics-and-qrels/qrels.covid-round1.txt runs/anserini.covid-r2.abstract.qdel.bm25.txt | egrep '(ndcg_cut_10 |recall_1000 )'
+
+wget https://www.dropbox.com/s/di27r5o2g5kat5k/lucene-index-cord19-full-text-2020-05-01.tar.gz
+tar xvfz lucene-index-cord19-full-text-2020-05-01.tar.gz
+
+target/appassembler/bin/SearchCollection -index lucene-index-cord19-full-text-2020-05-01 \
+ -topicreader Covid -topics src/main/resources/topics-and-qrels/topics.covid-round2.xml -topicfield query+question -removedups \
+ -bm25 -hits 10000 -output runs/anserini.covid-r2.full-text.qq.bm25.txt -runtag anserini.covid-r2.full-text.qq.bm25.txt
+
+target/appassembler/bin/SearchCollection -index lucene-index-cord19-full-text-2020-05-01 \
+ -topicreader Covid -topics src/main/resources/topics-and-qrels/topics.covid-round2-udel.xml -topicfield query -removedups \
+ -bm25 -hits 10000 -output runs/anserini.covid-r2.full-text.qdel.bm25.txt -runtag anserini.covid-r2.full-text.qdel.bm25.txt
+
+eval/trec_eval.9.0.4/trec_eval -c -M1000 -m all_trec src/main/resources/topics-and-qrels/qrels.covid-round1.txt runs/anserini.covid-r2.full-text.qq.bm25.txt | egrep '(ndcg_cut_10 |recall_1000 )'
```

Comment:
nbd but any reason to use egrep to filter `all_trec` results rather than just `-m ndcg_cut.10 -m recall.1000`?

Reply:
I used Ellen's exact invocation to signal conformance to official NIST settings. The egrep is just so I don't have to manually skim the output.

Reply:
ah, makes sense

Reply:
... so that people not familiar with arcane `trec_eval` parameters wouldn't be like, "wait, is that the correct evaluation setting?"